docs/reference/templating.rst

```diff
@@ -557,12 +557,12 @@ Location siblings
 - **Listing Location's siblings**
 
   This is done by calling the method ``getSiblings()``, also available as ``siblings()`` in
-  Twig. It returns an array of children Locations and optionally accepts maximum number of items
+  Twig. It returns an array of siblings Locations and optionally accepts maximum number of items
```

Comment:
`sibling`, without the `s`

Otherwise looks good :+1:

Reply:
You are correct!